src/components/App/App.tsx

```diff
@@ -6,48 +6,14 @@ import Lobby from "../Lobby/Lobby";
 import JoinGameForm from "../JoinGameForm/JoinGameForm";
 import Game from "../Game/Game";
 import Stats from "../Stats/Stats";
-// import { getGame, getPlayer, getStats } from '../Util/fetchCalls'
 import { useState } from 'react';
 import ErrorPage from "../ErrorPage/ErrorPage";
 
 
 function App() {
   const [game, setGame] = useState({});
   const [players, setPlayers] = useState([]);
-  //create questions state and consider also storing in session storage
```

Comment:
Removed the comment to tidy the code, but I'm wondering about storing a separate `questions` object in the `App` state and sessionStorage for backup since we can't retrieve them again from the BE.